cmd/thv-operator/pkg/registryapi/config.go

```diff
@@ -0,0 +1,124 @@
+package registryapi
+
+import (
+	"fmt"
+
+	serverconfig "github.com/stacklok/toolhive-registry-server/pkg/config"
+	mcpv1alpha1 "github.com/stacklok/toolhive/cmd/thv-operator/api/v1alpha1"
+)
+
+// ConfigManager provides methods to build registry server configuration from MCPRegistry resources
+type ConfigManager interface {
+	BuildConfig(mcpRegistry *mcpv1alpha1.MCPRegistry) (*serverconfig.Config, error)
+}
+
+// NewConfigManager creates a new instance of ConfigManager
+func NewConfigManager() ConfigManager {
+	return &configManager{}
+}
+
+type configManager struct{}
+
+const (
+	// RegistryJSONFilePath is the file path where the registry JSON file will be mounted
+	RegistryJSONFilePath = "/etc/registry/registry.json"
+)
+
+func (*configManager) BuildConfig(mcpRegistry *mcpv1alpha1.MCPRegistry) (*serverconfig.Config, error) {
```

Comment:
IIUC, we do not build the SyncPolicy and Filter fields of the returned serverconfig.Config